src/d3q27_cumulant/Dynamics.R

```diff
@@ -32,3 +34,30 @@ AddNodeType("Smagorinsky", "LES")
 AddNodeType("Stab", "ENTROPIC")
 AddNodeType("SymmetryY", "BOUNDARY")
 AddNodeType("SymmetryZ", "BOUNDARY")
+AddNodeType("Tinlet","BOUNDARY")
```

Comment:
Make it WVelocityTurbulent for now. But at the end, I think we don't need a separate boundary node type for that.